cmd/install.go

```diff
@@ -19,6 +19,7 @@ type installCmd struct {
 	rollback          bool
 	service           string
 	taskDefinitionArn string
+	timeout           int
```

Comment:
I would actually make this a time.Duration. 
you can then do things like time.Duration.Parse and it does things like 10m 5s 10h etc

Reply:
not having a unit associated with a time makes it hard to reason about

Reply:
I feel hesitant about having to explain to the user the GoLang string duration format strings, and dealing with ms/us/ns which I really don't care about and have no way to enforce. I feel it's simple just to say it's all seconds.